codechecker/CodeChecker.py

```diff
@@ -335,6 +335,38 @@ def signal_handler(sig, frame):
         debug_parser.set_defaults(func=arg_handler.handle_debug)
 
         # --------------------------------------
+        # plist parser
+        plist_parser = subparsers.add_parser('plist',
+                                             formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+                                             help='Parse plist files in the given directory.')
+        plist_parser.add_argument('-w', '--workspace', type=str,
+                                  dest="workspace",
+                                  default=default_workspace(),
+                                  help=workspace_help_msg)
+        plist_parser.add_argument('-n', '--name', type=str,
+                                  dest="name", required=True,
+                                  default=argparse.SUPPRESS,
+                                  help='Name of the analysis.')
+        plist_parser.add_argument('-d', '--directory', type=str,
+                                  dest="directory", required=True,
+                                  help='Path of a directory containing plist files to parse.')
+        plist_parser.add_argument('-j', '--jobs', type=int, dest="jobs",
+                                  default=1, required=False,
+                                  help='''\
+Number of jobs. Start multiple processes for faster analisys''')
```

Comment:
We could use global help messages for the arguments which are used at multiple places. Later modification will be easier.